proj-xcode/PowerAuthCore/PowerAuthCoreSession.mm

```diff
@@ -192,6 +192,12 @@ - (PowerAuthCoreProtocolVersion) protocolVersion
     return (PowerAuthCoreProtocolVersion) _session->getProtocolVersion();
 }
 
+- (NSDictionary<NSString*, NSObject*>*) lastUserInfo
+{
+    [self requireReadAccess:nil];
+    return cc7::objc::JsonValueToObjC(_session->lastUserInfo());
```

Comment:
Here's unhandled exception in case activation is not present. I think we can simplify this and annotate `lastUserInfo()` method with `noexcept`.